soil_id/global_soil.py

```diff
@@ -506,193 +566,173 @@ def aggregated_data_layer(data, column_name):
                 "ec": "ds/m",
             },
         },
-        "soilList": output_SoilList,
+        # "AWS_PIW90": aws_PIW90,
+        # "Soil Data Value": var_imp,
+        "soilList": output_SoilList_cleaned,
     }
 
+    return SoilListOutputData(
+        soil_list_json=soil_list_json,
+        rank_data_csv=soilIDRank_output_pd.to_csv(index=None, header=True),
+        map_unit_component_data_csv=mucompdata_cond_prob.to_csv(index=None, header=True),
+    )
+
 
 ##############################################################################################
 #                                   rankPredictionGlobal                                     #
 ##############################################################################################
-def rankPredictionGlobal(
-    lon, lat, soilHorizon, horizonDepth, rfvDepth, lab_Color, bedrock, cracks, plot_id=None
+def rank_soils_global(
```

Comment:
issue: this should be called rank_soils() to match the us version. this module is namespaced and doesn't need a prefix.